api/graphql/mutations.js

```diff
@@ -83,18 +83,16 @@ export function updateMembership (userId, { id, data }) {
 
   return Membership.query().where({id, user_id: userId})
   .update(whitelist)
-  .returning('id')
-  .then(ids => Membership.where('id', ids[0]).fetch())
+  .then(() => ({success: true}))
```

Comment:
why go from returning the membership to returning success true? is the front-end prepared for this change?

As I read further I interpret this is because we're prepping mutations to mostly just return this GenericResult type?